### DISCUSSION TITLE
GTK GUI: What is "Status Open/Closed". Where to configure it?

### DISCUSSION THREAD

**Discussion by marek141017** [baresip/baresip#3137] [General]
Hello
There is option in GTK GUI "Status Open/Closed"
I need to make it Open by default.

It looks like I can not find it in configuration
Please advice

Thank you

**Comment by mbattista** [comment]
Hi,

the status open/closed is your current presence.
As default it will take the current presence of the presence module.

GTK does not offer to set all the accounts to status open on startup and as far as I can see neither does the presence module.

**Comment by marek141017** [comment]
Thank you.
I will use command line version.
This one is exactly what I was looking for.
Pulseaudio is pretty good for removing echo. Do not undestand why Ubuntu has not compliled webrtc-aec module. 

Thank you very much for this application.
